src/Layouts/Partials/AlignItemsPanel.vala

```diff
@@ -54,14 +68,10 @@ public class Akira.Layouts.Partials.AlignItemsPanel : Gtk.Grid {
         halign = Gtk.Align.FILL;
         get_style_context ().add_class ("alignment-box");
 
-        var alignment_box = new Gtk.Grid ();
-        alignment_box.halign = Gtk.Align.CENTER;
-        alignment_box.hexpand = true;
-
         foreach (var item in ALIGN_ITEMS_PANEL_BUTTONS) {
             switch (item.type) {
                 case "sep":
-                    alignment_box.attach (new Gtk.Separator (Gtk.Orientation.VERTICAL),
+                    attach (new Gtk.Separator (Gtk.Orientation.VERTICAL),
```

Comment:
Let's halign the separators to the center, this will play better with the column_homogeneous:
```
var separator = new Gtk.Separator (Gtk.Orientation.VERTICAL);
separator.halign = Gtk.Align.CENTER;
attach (separator, current_button_column++, 0, 1, 1 );
```

Reply:
OK